core/src/main/java/org/apache/brooklyn/util/core/logbook/LogBookQueryParams.java

```diff
@@ -20,12 +20,28 @@
 
 import java.util.List;
 
+/**
+ * The logbook query parameters.
+ */
 public class LogBookQueryParams {
+
+    /** The number of log items to query. Note, one log item can be a multi-line one, e.g. a stacktrace */
     private Integer numberOfItems;
+
+    /** The indicator whether to return logs in reverse order */
     private Boolean reverseOrder;
+
+    /** The log levels: INFO, FATAL, ERROR, DEBUG or WARNING */
     private List<String> levels;
-    private String initTime;
-    private String finalTime;
+
+    /** The date-time to look log items from */
+    private String dateTimeFrom;
```

Comment:
why isn't this a date?

Reply:
Not my choice but a valid point. Looks like LocalDateTime is more appropriate, timezone is not required. What do you think @duncangrant ?

Reply:
Calendar input does take timezone into account though.
<img width="1173" alt="Screenshot 2021-07-06 at 18 35 50" src="https://user-images.githubusercontent.com/81319331/124643421-fe66ca80-de88-11eb-9bfe-dc4dce657452.png">